cluster/azure/README.md

```diff
@@ -178,7 +178,7 @@ In production scenarios, storing the state file on a local file system is not de
 
 ```bash
 terraform {
-   backend “azure” {
+   backend "azurerm" {
```

Comment:
you're replacing `azure` with `azurerm`?  Is that on purpose

Reply:
I did it on purpose. Could still be wrong.

When I used "azure" as the docs suggest Terraform complained. Then I found an example here https://github.com/microsoft/bedrock/blob/6e0bb1d97ba2c47a62181108c607b9f9528e42ae/cluster/environments/azure-common-infra/main.tf#L2 where "azuurerm" was being used instead. That worked...so I was just assuming the docs hadn't been updated.

Reply:
thanks @frodopwns  for the clarification.  I'll leave it to @evanlouie @mtarng to double check if it's an issue with outdated doc